Formula/now-cli.rb

```diff
@@ -0,0 +1,26 @@
+require "language/node"
+
+class NowCliBeta < Formula
+  desc "The command-line interface for Now"
+  homepage "https://zeit.co/now"
+  url "https://registry.npmjs.org/now/-/now-16.1.0.tgz"
+  sha256 "c089f0add1a4652f01f945d7fed75c811e6268b7c1c180e96876581480f59242"
+
+  depends_on "node"
+
+  def install
+    rm Dir["dist/{*.exe,xsel}"]
+    inreplace "dist/index.js" do |s|
+      s.gsub! /(\w+).(\w+)=getUpdateCommand/, "\\1.\\2=async()=>'Please run `brew upgrade now-cli` to update Now CLI.'"
+      s.gsub! '"now update"', '"brew upgrade now-cli"'
+    end
```

Comment:
```ruby
    inreplace "dist/index.js", "t.default=getUpdateCommand",
                               "t.default=async()=>'brew upgrade now-cli'"
```

This should be enough to update the update-command.

Reply:
I want to make sure it will continue working even if the exported name changed in feature.

Reply:
But they changed the `getUpdateCommand` function in https://github.com/zeit/now-cli/commit/05c2f69523be6f23717d91cd5124221bbd73fdca#diff-9e2823661fa83a88b9ec7ba4be79faf2 to only return the command and not all the boilerplate text around it, so this `inreplace` needs to be updated for it.
Also it's moved to it's own file now it it's there the single default export, so assuming that it will stay this way should be pretty safe.

For what was you `s.gsub! '"now update"', '"brew upgrade now-cli"'` intended?

Reply:
OK, i didn't notice that commit before, I will change the command text soon, but `gsub` isn't bad, is it?

The next replacement is for `now upgrade`, it will output something like `Are you trying to update now-cli? try "now update"`.

Reply:
Yeah, but `now update` would just print:
```sh
> Please run `brew upgrade now-cli` to update Now CLI
```
 which is totally fine for us. Just drop this and only replace the `getUpdateCommand`. And since we are doing only 1 replace than a `inreplace` block with multiple `s.gsub!` isn't necessary anymore and you can just use a normal `inreplace`.

BTW: Also your regex wouldn't survive a exported name change from `getUpdateCommand` to something else either. The first part is just the webpacked default export and given it's in it's own `get-update-command.ts` now it's quite unlike to change to something else than the default export.

Reply:
Then the user will run `now update` and then `brew upgrade now-cli` again, I don't think it's good.

`Also your regex wouldn't survive a exported name change from getUpdateCommand to something else either.`

I don't understand why? For example: `t.getUpdateCommand= getUpdateCommand` it should work.

Reply:
`now update` does nothing more than printing the correct update command to the console and it does it pretty fine once we've patched it in `getUpdateCommand`. There is no need to avoid it since it won't run something again, it just hints the user how to update itself based on how it was installed.

The thing which broke the regex earlier was the rename from getUp*grade*Command to getUp*date*Command, which wouldn't be covered by your regex. Your regex would just cover moving it from a default export to a non-default named export, which is quite unlikely because it lives now in its own file serving only the single purpose to get the installation method dependent update command.